docs/source/release_notes.rst

```diff
@@ -4,6 +4,211 @@
 Release Notes
 *************
 
+pyMOR 2020.1 (July ??, 2020)
+----------------------------
+We are proud to announce the release of pyMOR 2020.1! Highlights of this release
+are support for non-intrusive model order reduction using artificial neural networks,
+the subspace accelerated dominant pole algorithm (SAMDP) and the implicitly restarted
+Arnoldi method for eigenvalue computation. Parameter handling in pyMOR has been
+simplified, and a new series of hands-on tutorials helps getting started using pyMOR
+more easily.
+
+Over 600 single commits have entered this release. For a full list of changes
+see `here <https://github.com/pymor/pymor/compare/2019.2.x...2020.1.x>`__.
+
+pyMOR 2020.1 contains contributions by Linus Balicki, Tim Keil, Hendrik Kleikamp
+and Luca Mechelli. We are also happy to welcome Linus as a new main developer!
+See `here <https://github.com/pymor/pymor/blob/master/AUTHORS.md>`__ for more
+details.
+
+
+Release highlights
+^^^^^^^^^^^^^^^^^^
+
+Model order reduction using artificial neural networks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With this release, we introduce a simple approach for non-intrusive model order
+reduction to pyMOR that makes use of artificial neural networks
+`[#1001] <https://github.com/pymor/pymor/pull/1001>`_. The method was first
+described in [HU18]_ and only requires being able to compute solution snapshots of
+the full-order |Model|. Thus, it can be applied to arbitrary (nonlinear) |Models| even when no
+access to the model's |Operators| is possible.
+
+Our implementation internally wraps `PyTorch <https://pytorch.org>`_ for the training and evaluation of
+the neural networks. No knowledge of PyTorch or neural networks is required to apply the method.
+
+
+New system analysis and linear algebra algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+So far, computing poles of an |LTIModel| was only supported by its
+:meth:`~pymor.models.iosys.LTIModel.poles` method which uses a dense eigenvalue
+solver and converts the operators to dense matrices.
+The new version adds the :meth:`~pymor.algorithms.eigs.eigs` method
+(`[#880] <https://github.com/pymor/pymor/pull/880>`_) for computing a few
+smallest/largest eigenvalues (similar to `scipy.sparse.linalg.eigs`, except
+working for generic operators).
+Furthermore, a new addition is the :meth:`~pymor.algorithms.samdp.samdp` method
+(`[#834] <https://github.com/pymor/pymor/pull/834>`_) which takes into account
+the input and output operators of an |LTIModel| to compute its dominant poles.
```

Comment:
The new version still confuses me. How about:

```suggestion
The new :meth:`~pymor.algorithms.eigs.eigs` method
(`[#880] <https://github.com/pymor/pymor/pull/880>`_) computes 
smallest/largest eigenvalues of an arbitary linear real |Operator|
using the implicitly restarted Arnoldi method [RL95]_. It can also
be used to solve generalized eigenvalue problems.

So far, computing poles of an |LTIModel| was only supported by its
:meth:`~pymor.models.iosys.LTIModel.poles` method, which uses a dense eigenvalue
solver and converts the operators to dense matrices.
The new :meth:`~pymor.algorithms.samdp.samdp` method
(`[#834] <https://github.com/pymor/pymor/pull/834>`_) implements the
subspace accelerated dominant pole (SAMDP) algorithm  [RM06]_,
which can be used to compute the dominant poles operators of an 
|LTIModel| with arbitrary (in particular sparse) system |Operators|
without relying on dense matrix operations.
```

Reply:
Yes, that sounds better. I forgot about other uses of `eigs`...